content/en/security/cspm/frameworks_and_benchmarks.md

```diff
@@ -38,31 +37,33 @@ Each [OOTB rule][1] maps to one or more controls within a compliance standard or
 
 **Some [CIS Kubernetes Benchmark][5] detection rules only apply to self-hosted Kubernetes clusters.
 
-**Note**: Datadog CSPM provides you with visibility into whether your resources are configured in accordance with certain detection rules. Datadog's OOTB detection rules address various regulatory frameworks, benchmarks, and standards (“Security Posture Frameworks”). Datadog CSPM does not provide an assessment of your actual compliance with any Security Posture Framework, and the OOTB rules may not address all configuration settings that are relevant to the Security Posture Frameworks. To be clear, just because your resources pass the OOTB Rules does not mean that you are meeting all the requirements under any particular Security Posture Framework. Datadog is not providing legal or compliance advice or guidance, and it is recommended that you use Datadog CSPM in consultation with your legal counsel or compliance experts.
+Datadog also provides Datadog Security Labs, a set of recommendations developed by Datadog internal security experts. Based on common cloud security risks we have observed at Datadog, this ruleset aims to help users new to cloud security easily remediate high-impact misconfigurations across their cloud environments.
```

Comment:
Datadog Security Labs has been renamed `Essential Cloud Security Controls`

Reply:
Thanks for clarifying. I'll fix it in the PR.